### ISSUE TITLE
Panorama Certificate Import Error

### ISSUE THREAD

**Issue by fh-dfarrar** [PaloAltoNetworks/pango#95]
## Describe the bug

This is referencing the bug outlined here:
https://github.com/PaloAltoNetworks/terraform-provider-panos/issues/329

Terraform panos_certificate_import works on firewalls but not on Panorama, while the certificate imports fine via the Panorama GUI.

## Expected behavior

panos_certificate_import should be able to import a certificate that the GUI finds valid.

## Current behavior

Error message when Terraform try to import the certificate:
> Error: Import of panorama.domain.com failed. Failed to extract certificate

## Possible solution
This fixed PEM format certs, not sure about PKCS12 certs yet but will test.  It seems "certificate" was being referenced instead of the private key, which is why the TF provider could import the public certificate with the first API call but the 2nd API call error'd out while importing the private key.

pango/dev/certificate/pano.go
line 114
change "certificate" to "private-key" as such:
`	_, err = c.ns.Client.Import("private-key", cert.PrivateKey, cert.PrivateKeyFilename, "file", timeout, ex, nil)`

## Steps to reproduce

Basically using the demo code:

`resource "panos_certificate_import" "example" {
    name = "testnew1234"
    template   = panos_panorama_template.demo_template.name
    pem {
        certificate = file("cert.pem")
        private_key = file("key.pem")
        passphrase = "examplepass"
    }

    lifecycle {
        create_before_destroy = true
    }
}`

## Your Environment

- Version used: PanOS 10.2.3
- Link to your project: https://github.com/fh-dfarrar/pango

**Comment by shinmog** [comment]
Fixed by #96
